napari_animation/viewer_state.py

```diff
@@ -32,7 +34,11 @@ def from_viewer(cls, viewer: napari.viewer.Viewer):
         }
         for layer_attributes in layers.values():
             layer_attributes.pop("metadata")
-            layer_attributes.pop("properties", None)
+            # the following can't be set as attributes
+            layer_attributes.pop("property_choices", None)
+            layer_attributes.pop("ndim", None)
+            layer_attributes.pop("colormaps_dict", None)
```

Comment:
Would be better if we automatically detected unsettable stuff instead, so we don't have to maintain a whitelist and reduce issues with new attributes. We can try/catch for `AttributeError` and simply do nothing in that case.